Library/Formula/rust.rb

```diff
@@ -31,8 +31,11 @@ class Rust < Formula
     sha256 "04f68ef26b4ca636b35be1d17ad2b641727bbc10cec52582a9e3a64b5784dbeb" => :mavericks
   end
 
+  option "with-llvm", "Build with brewed LLVM"
```

Comment:
I think actually `with-llvm` makes more sense and we should move more formulae to use that approach.